pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -341,39 +343,81 @@ func (o *Options) Run() error {
 				fmt.Fprintf(o.Out, "Upstream: %s\n", cv.Spec.Upstream)
 			}
 			if len(cv.Status.Desired.Channels) > 0 {
-				fmt.Fprintf(o.Out, "Channel: %s (available channels: %s)\n", cv.Spec.Channel, strings.Join(cv.Status.Desired.Channels, ", "))
+				fmt.Fprintf(o.Out, "Channel: %s (available channels: %s)\n\n", cv.Spec.Channel, strings.Join(cv.Status.Desired.Channels, ", "))
 			} else {
-				fmt.Fprintf(o.Out, "Channel: %s\n", cv.Spec.Channel)
+				fmt.Fprintf(o.Out, "Channel: %s\n\n", cv.Spec.Channel)
 			}
 		}
 
 		if len(cv.Status.AvailableUpdates) > 0 {
-			fmt.Fprintf(o.Out, "Available Updates:\n\n")
-			w := tabwriter.NewWriter(o.Out, 0, 2, 1, ' ', 0)
-			fmt.Fprintf(w, "VERSION\tIMAGE\n")
+			fmt.Fprintf(o.Out, "Recommended updates:\n\n")
+			// set the minimal cell width to 14 to have a larger space between the columns for shorter versions
+			w := tabwriter.NewWriter(o.Out, 14, 2, 1, ' ', 0)
+			fmt.Fprintf(w, "  VERSION\tIMAGE\n")
 			// TODO: add metadata about version
-			sortSemanticVersions(cv.Status.AvailableUpdates)
+			sortReleasesBySemanticVersions(cv.Status.AvailableUpdates)
+			reverseReleases(cv.Status.AvailableUpdates)
```

Comment:
Instead of sorting and then reversing, can we just flip the sense of the sort?  `sortReleasesBySemanticVersions` is an internal helper with this one use-case, and we can just adjust the godocs to say it's becoming `decreasing`.  That would allow this call site to say "sort these however we like them sorted" without having to worry about whether it was increasing or decreasing.

Reply:
Yes, that seems very reasonable. 
Fixed in the new commits ae4725e and d747e5b (see: https://github.com/openshift/oc/pull/961#issuecomment-968759024).